welcome/oke_about.adoc

```diff
@@ -216,278 +216,120 @@ deployments.
+| Disconnected Installation | Included | Included | N/A
+| Red Hat Enterprise Linux (RHEL) or CoreOS entitlement | Included | Included | N/A
+| Existing RHEL manual attach to cluster (BYO) | Included | Included | N/A
+| CRIO Runtime | Included | Included | N/A
+| Over the Air Smart Upgrades and Operating System (CoreOS) Management | Included | Included | N/A
+| Enterprise Secured Kubernetes | Included | Included | N/A
+| Kubectl and `oc` automated command line | Included | Included | N/A
+| Auth Integrations, RBAC, SCC, Multi-Tenancy Admission Controller | Included | Included | N/A
+| Operator Lifecycle Manager (OLM) | Included | Included | N/A
+| Administrator web console | Included | Included | N/A
+| OpenShift Virtualization | Included | Included | OpenShift Virtualization Operator
+| Compliance Operator provided by Red Hat | Included | Included | Compliance Operator
+| File Integrity Operator | Included | Included | File Integrity Operator
+| Gatekeeper Operator | Included | Included | Gatekeeper Operator
+| Klusterlet provided by Red Hat | Included | Included | N/A
+| Kube Descheduler Operator provided by Red Hat | Included | Included | Kube Descheduler Operator
+| Local Storage provided by Red Hat | Included | Included | Local Storage Operator
+| Node Feature Discovery provided by Red Hat | Included | Included | Node Feature Discovery operator
+| Performance Add-on Operator | Included | Included | Performance Add-on Operator
+| PTP Operator provided by Red Hat | Included | Included | PTP Operator
+| Service Telemetry Operator provided by Red Hat | Included | Included | Service Telemetry Operator
+| SR-IOV Network Operator | Included | Included | SR-IOV Network Operator
+| Vertical Pod Autoscaler | Included | Included | Vertical Pod Autoscaler
+| Cluster Monitoring (Prometheus) | Included | Included | Cluster Monitoring
+| Device Manager (for example, GPU) | Included | Included | N/A
+| Log Forwarding (with fluentd) | Included | Included | Cluster Logging Operator (for log forwarding with fluentd)
+| Telemeter and Insights Connected Experience | Included | Included | N/A
+| OpenShift Cloud Manager(OCM) SaaS Service | Included | Included | N/A
+| OVS and OVN SDN | Included | Included | N/A
+| HAProxy Ingress Controller | Included | Included | N/A
+| {rh-openstack-first} Kuryr Integration | Included | Included | N/A
+| Ingress Cluster-wide Firewall | Included | Included | N/A
+| Egress Pod and Namespace Granular Control | Included | Included | N/A
+| Ingress Non-Standard Ports | Included | Included | N/A
+| Multus and Available Multus Plugins | Included | Included | N/A
+| Network Policies | Included | Included | N/A
+| IPv6 Single and Dual Stack | Included | Included | N/A
+| CNI Plugin ISV Compatibility | Included | Included | N/A
+| CSI Plugin ISV Compatibility | Included | Included | N/A
+| RHT and IBM middleware à la carte purchases (not included in {product-title} or {oke}) | Included | Included | N/A
```

Comment:
I'm confused about what this row is supposed to indicate. Am I misreading, or are these purchases both included and not for both OCP and OKE?